src/opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -1184,7 +1184,7 @@ bool parseState( ParserState& parserState, const Parser& parser ) {
         if (dataFileName[0] == '/')
             data_file = Opm::filesystem::canonical(dataFileName).string();
         else
-            data_file = Opm::filesystem::relative( Opm::filesystem::canonical(dataFileName) );
+            data_file = Opm::filesystem::proximate( Opm::filesystem::canonical(dataFileName) );
 
```

Comment:
I think you need to remove filesystem. Opm::proximate

Reply:
OK - have you tried? The current PR works on my machine - for what it is worth

Reply:
you have to use Opm::proximate. You are using the one exposed in the filesystem namespace which is not available with the ts.

Reply:
If I use Opm::proximate everything build fine

Reply:
OK - updated